### DISCUSSION TITLE
(Official) Apollo Server in rust

### DISCUSSION THREAD

**Discussion by ghost** [apollographql/apollo-rs#292] [Ideas]
Not sure where to route this question. We can see the Apollo team launching various graphQL related 'rust' repos - are there any plans to launch an 'official' Apollo Server (graphQL) in rust (i.e. by the Apollo team)?

**Comment by lrlna** [comment]
Hi @dotjon0! Thanks for asking, this is a good question. The TL;DR is there are no plans to have an official apollo rust server in the near future. 

The ever-so-slightly longer answer is that this was brought up internally a few times, but at the moment we haven't necessarily found a need for it in our own work, nor have a lot of our users asked for this - you're one of the first. This may change in the future if we see the need arise (never say never)!

With that said, if anyone reads this and is looking for an apollo rust server, you may use this issue to note down your interest.

**Comment by ghost** [comment]
The compute savings and memory safety security benefits of rust make a clear case for GraphQL in Rust... Maybe a collaboration with https://github.com/graphql-rust/juniper is an idea... We love Apollo Server, with rust involved this makes an incredible combination... Thanks for your feedback @lrlna

**Comment by ghost** [comment]
We have publicised this via the official Rust community https://users.rust-lang.org/t/apollo-server-in-rust-lang-get-your-votes-in/80991

**Reply by lrlna** [reply]
Thank you so much!

**Comment by ghost** [comment]
@lrlna Juniper traction is growing... On Github, Juniper has 36% of the quantity of 'stars', 19% of the 'forks' and 27% 'watches' VS Apollo Server - see below. 

<img width="839" alt="Screenshot 2022-09-08 at 16 12 24" src="https://user-images.githubusercontent.com/108075644/189159366-2e7b77cb-0ca0-4858-9970-687f785239ff.png">

<img width="829" alt="Screenshot 2022-09-08 at 16 12 58" src="https://user-images.githubusercontent.com/108075644/189159479-9ac71cfb-1c3e-41d5-9aec-3f5fa286e0ba.png">
